conf/igenomes.config

```diff
@@ -0,0 +1,440 @@
+/*
```

Comment:
Is it right, that the whole file is new? You didn't have an igenomes file before?

Reply:
no, because it does not use igenomes but pull specific reference data for BCR / TCR